deps.ts

```diff
@@ -1 +1,2 @@
 export * as lsp from "https://cdn.skypack.dev/vscode-languageserver-types?dts";
+export { default as stripJSONcomments } from "https://cdn.skypack.dev/strip-json-comments?dts";
```

Comment:
I don't know what the inclusion of `?dts` does. Sorry if it's not necessary.

Reply:
?dts is a way of telling Skypack to provide Typescript typings when transforming this NPM package to ESM.